frontend/src/components/Plan/hook/fetchPlanData.ts

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react'
+import { loadingStatus } from '../../../utils/loadingStatus'
+import axios from 'axios'
+
+export const fetchPlanData = (planIdParam: any): any => {
```

Comment:
try to avoid explicitly typing `any`. this looks like It should be 1) an async fn and 2) have a real typing